models/organization/team_repo.go

```diff
@@ -81,5 +81,6 @@ func GetTeamsWithAccessToRepo(ctx context.Context, orgID, repoID int64, mode per
 		Join("INNER", "team_repo", "team_repo.team_id = team.id").
 		And("team_repo.org_id = ?", orgID).
 		And("team_repo.repo_id = ?", repoID).
+		OrderBy("name").
```

Comment:
unrelated?

Reply:
No, this is sorted by team name in the review list

Reply:
Is this field team sort, if there is a team list
![image](https://user-images.githubusercontent.com/1255041/174693612-c81634f0-141c-468d-8283-bb3b040428b9.png)